development-defaults.ini

```diff
@@ -68,21 +72,53 @@ attendee_badge = 3000, 29999
 one_day_badge = 30000, 39999
 
 
```

Comment:
Ooh, that part will be a problem. We'll have to override that in the AC plugin if we do merge the branches again.